src/server.ts

```diff
@@ -0,0 +1,45 @@
+import * as express from "express";
+import * as session from "express-session";
+import * as bodyParser from "body-parser";
+import * as cookieParser from "cookie-parser";
+import {appRouter} from "./routes/routes"
+import * as nunjucks from 'nunjucks'
+
+
+
+/**
+ * Create Express server.
+ */
+const app = express();
```

Comment:
I have seen this setup so many times giving `server.ts` so many reasons for change.

May I ask you to think about splitting it into smaller classes? I see the following responsibilities:

1. initialisation of express framework
2. initialisation of templating engine (nunjucks)

I'd rather see server.ts be a place where things are put together while initialisation of components is delegated to other files. Place where it is easy to spot application routes and port application launches on.